index.js

```diff
@@ -64,22 +64,26 @@ module.exports = {
         const releaseName = `${appName}@${this.readConfig('revisionKey')}`;
 
         this.log('SENTRY: Deleting release...');
-        this.sentryCliExec(`releases delete ${releaseName}`);
+        this.sentryCliExec(`delete ${releaseName}`);
         this.log('SENTRY: Release deleted!');
       },
 
-      sentryCliExec(command) {
+      sentryCliExec(params) {
```

Comment:
We should get back to command :/ Params would an array of object of options.

Reply:
If this.sentryCliExec('release', `new ${releaseName}`); style ok to you. I'll change the signature with as sentryCliExec(command, params)?